net-kit/autogen.kit.d/base.yml

```diff
@@ -2627,4 +2627,189 @@ gitlab_freedesktop_dirlisting:
+            replace: ''
+          - kind: string
+            match: 'b1'
+            replace: ''
+          - kind: string
+            match: 'b2'
+            replace: ''
+          - kind: string
+            match: 'b3'
+            replace: ''
+          - kind: string
+            match: 'b4'
+            replace: ''
+          - kind: string
+            match: 'rc1'
+            replace: ''
+          - kind: string
+            match: 'rc2'
+            replace: ''
+        vars:
+          desc: library for developing printing features, split out of cups-filters
+          license: Apache-2.0
+          slot: '0'
+          homepage: https://github.com/OpenPrinting/libcupsfilters
+          inherit:
+              - libtool
+          iuse: dbus exif jpeg +poppler +postscript png test tiff
+          restrict: '!test? ( test )'
+          rdepend: |
+              >=app-text/qpdf-8.3.0:=
+              media-libs/fontconfig
+              media-libs/lcms:2
+              >=net-print/cups-2
+              !<net-print/cups-filters-2.0.0
+
+              exif? ( media-libs/libexif )
+              dbus? ( sys-apps/dbus )
+              jpeg? ( media-libs/libjpeg-turbo:= )
+              postscript? ( app-text/ghostscript-gpl[cups] )
+              poppler? ( >=app-text/poppler-0.32:=[cxx] )
+              png? ( media-libs/libpng:= )
+              tiff? ( media-libs/tiff:= )
+          depend: '${RDEPEND}'
+          bdepend: |
+              >=sys-devel/gettext-0.18.3
+              virtual/pkgconfig
+              test? ( media-fonts/dejavu )
+          body: |
+            src_prepare() {
+              default
+
+              # respect --as-needed
+              elibtoolize
+            }
+
+            src_configure() {
+              local myeconfargs=(
+                --enable-imagefilters
+                --localstatedir="${EPREFIX}"/var
+                --with-cups-rundir="${EPREFIX}"/run/cups
+                --disable-mutool
+                $(use_enable exif)
+                $(use_enable dbus)
+                $(use_enable poppler)
+                $(use_enable postscript ghostscript)
+                $(use_with jpeg)
+                $(use_with png)
+                $(use_with tiff)
+              )
+
+              econf "${myeconfargs[@]}"
+            }
+
+            src_install() {
+              default
+              find "${ED}" -name '*.la' -delete || die
+            }
 
+libppd:
+  generator: builtin-github
+  template:
+    engine: helm
+
+  defaults:
+    files_dir: "{{ .Values.pn }}"
+    template: ../../templates/simple.tmpl
+    category: net-print
+    github:
+      user: OpenPrinting
+      repo: libppd
+    assets:
+      - name: '{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+        url: 'https://github.com/OpenPrinting/{{ .Values.pn }}/releases/download/{{ .Values.version }}/{{ .Values.pn }}-{{ .Values.version }}.tar.xz'
+
+  packages:
+    - libppd:
+        transform:
```

Comment:
Maybe can we disable injection of the `rc|beta` releases? This could be done as exclude option and cleanup the trasforms.